mautic/mautic
# Add created by to report and segments

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
We need use **created by** on leads in segment filters and in contacts reports.  This PR added select field with users 



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create contact
3. Create segment with filter Created By user who create contact
4. Rebuild segment
5. You should see created contact in segment
6. Create contacts report
7. Add column Created By
8. Use created by in filter and filter by user who create contact
9. Check report
10. You should see your contact  there
